Sources/StreamChatUI/ChatMessageList/Attachments/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -7,55 +7,56 @@ import UIKit
+        open var didTapOnAttachment: ((ChatMessageFileAttachment) -> Void)?
+        
+        /// Label which shows name of the file, usually with extension (file.pdf)
+        open private(set) lazy var fileNameLabel = UILabel()
             .withoutAutoresizingMaskConstraints
             .withBidirectionalLanguagesSupport
             .withAdjustingFontForContentSizeCategory
-
-        public private(set) lazy var fileSizeLabel = UILabel()
+        
+        /// Label indicating size of the file.
+        open private(set) lazy var fileSizeLabel = UILabel()
             .withoutAutoresizingMaskConstraints
             .withBidirectionalLanguagesSupport
             .withAdjustingFontForContentSizeCategory
-
-        public private(set) lazy var loadingIndicator = components
+        
+        /// Animated indicator showing progress of uploading of a file.
+        open private(set) lazy var loadingIndicator = components
             .messageList
             .messageContentSubviews
             .attachmentSubviews
             .loadingIndicator
             .init()
             .withoutAutoresizingMaskConstraints
+        
+        /// imageView indicating action for the file attachment. (Download / Retry upload...)
+        open private(set) lazy var actionIconImageView = UIImageView().withoutAutoresizingMaskConstraints
 
-        public private(set) lazy var actionIconImageView = UIImageView()
-            .withoutAutoresizingMaskConstraints
-
-        public private(set) lazy var spinnerAndSizeStack: UIStackView = {
-            let stack = UIStackView(arrangedSubviews: [loadingIndicator, fileSizeLabel])
-            stack.axis = .horizontal
-            stack.spacing = UIStackView.spacingUseSystem
-            stack.alignment = .center
+        open private(set) lazy var mainContainerStackView: ContainerStackView = {
+            let stack = ContainerStackView(arrangedSubviews: [fileIconImageView, fileNameAndSizeStack, actionIconImageView])
             return stack.withoutAutoresizingMaskConstraints
         }()
-
-        public private(set) lazy var fileNameAndSizeStack: UIStackView = {
-            let stack = UIStackView(arrangedSubviews: [fileNameLabel, spinnerAndSizeStack])
-            stack.axis = .vertical
-            stack.alignment = .leading
-            stack.spacing = 3
+        
+        /// Stack containing loading indicator and label with fileSize.
+        open private(set) lazy var spinnerAndSizeStack: ContainerStackView = {
+            let stack = ContainerStackView(arrangedSubviews: [loadingIndicator, fileSizeLabel])
             return stack.withoutAutoresizingMaskConstraints
         }()
 
-        public private(set) lazy var fileIconImageView = UIImageView()
-            .withoutAutoresizingMaskConstraints
+        /// Stack containing file name and and the size of the file.
+        open private(set) lazy var fileNameAndSizeStack: ContainerStackView = {
```

Comment:
I will nit this myself: No need to use the brackets here and above... Just leftovers